test/index.test.js

```diff
@@ -188,25 +188,79 @@ describe('Index Tests', () => {
       path: '/index.html',
     });
 
-    scope.done();
+    stopScope.done();
+    outerScope.done();
+    innerScope.done();
     assert.strictEqual(result.statusCode, 207);
     assert.deepStrictEqual(result.body, [
       { status: 'ok', url: 'https://blog.adobe.com/index.html' },
+      { status: 'ok', url: 'https://blog.adobe.com/' },
+      { status: 'ok', url: 'https://blog.adobe.com/index' },
       { status: 'error', url: 'https://theblog--adobe.hlx.page/index.html' },
     ]);
   }).timeout(5000);
 
-  it('index function purges outer cdn and inner cdn', async () => {
+  [
+    ['/', 4, 3, 1], // also purge /index(.html) and /index.md
```

Comment:
This is super opaque. What about having a tuple of the input URL and a Set containing all the URLs that get a `PURGE` request, plus an optional set of all URLs getting an API call?

To make things easier when tests fail, add a description into the structure instead of just keeping it as a comment.

Reply:
> ... a Set containing all the URLs that get a `PURGE` request

Blocked by https://github.com/adobe/helix-pipeline/issues/985

Reply:
You can start with the test setup and we tweak the actual test/spec as we get clarity on adobe/helix-pipeline#985